src/test/java/tp04/metier/JourTest.java

```diff
@@ -57,4 +57,4 @@ protected void testConstructorDayIncorrectShouldFail() {
         Assertions.assertEquals(expectedMessage, currentMessage, "Expected error message");
 
     }
-}
+}
```

Comment:
Peux-tu ajouter un saut de ligne ap^rès `}` ?
```suggestion
}

```